kuma/users/admin.py

```diff
@@ -21,7 +21,8 @@ class UserAdmin(BaseUserAdmin):
 
 class UserBanAdmin(admin.ModelAdmin):
     fields = ('user', 'by', 'reason', 'is_active')
-    list_display = ('user', 'by', 'reason')
+    list_display = ('user', 'by', 'reason', 'is_active')
+    list_editable = ('is_active',)
```

Comment:
Having both `UserBan.is_active` and `User.is_active` that are essentially opposites of each other is very confusing here.